Libraries/Opc.Ua.Server/Configuration/ConfigurationNodeManager.cs

```diff
@@ -602,7 +603,11 @@ public void HasApplicationSecureAdminAccess(ISystemContext context, string _)
             ref byte[][] certificates)
         {
             HasApplicationSecureAdminAccess(context);
-
+            //No rejected store configured
+            if (m_rejectedStorePath == null)
+            {
```

Comment:
is there a need to initialize certificates to an empty byte[]?

Reply:
oops, done